### PR TITLE
Add addressbook-level1 contributors to Team.md

### PR DESCRIPTION
**Pull request by limmlingg**

### CODE REVIEW

File: docs/Team.md [hunk 1]
```diff
@@ -80,7 +80,8 @@ Area Lead for: UI [2016 Aug - 2017 May]
 
 #### [Leow Yijin](http://github.com/yijinl)
 <img src="images/LeowYijin.jpg" width="100"><br>
-Area Lead for: Model [2016 May - 2017 May]
+Area Lead for: Model [2016 May - 2017 May]<br>
+Main developer for the first version of the AddressBook-level1
```
Comment (Zhiyuan-Amos) [inline docs/Team.md]: Capital L for "level1" haha

**Comment by Zhiyuan-Amos** [comment]
On a side note prof @damithc, is there an error with me being the `Area Lead for: Logic [2017 Aug - ]`? I mean, I don't think I'm working on the `Logic` component and am likely to be the only developer (i.e I'm leading myself? Haha) after August. XD

**Comment by damithc** [comment]
> is there an error with me being the Area Lead for: Logic [2017 Aug - ]? I mean, I don't think I'm working on the Logic component and am likely to be the only developer (i.e I'm leading myself? Haha) after August. XD

That's fine. An area lead needs an area. You can use some other area if you wish e.g. GUI, Testing.

**Review by yamgent — COMMENTED** [review]
No other comments.

File: docs/Team.md [hunk 2]
```diff
@@ -80,7 +80,8 @@ Area Lead for: UI [2016 Aug - 2017 May]
 
 #### [Leow Yijin](http://github.com/yijinl)
```
Comment (yamgent) [inline docs/Team.md:81]: On an unrelated note (not related to this PR), this link is broken. Did the person changed his alias/deleted his account?

**Comment by limmlingg** [comment]
@Zhiyuan-Amos @yamgent hi

**Review by Zhiyuan-Amos — APPROVED** [review]

**Review by yamgent — APPROVED** [review]

**Comment by Zhiyuan-Amos** [comment]
https://github.com/AlexLMeow This looks like his account. When I did a quick Google search (https://www.google.com.sg/search?q=leow+yijin+github&rlz=1C5CHFA_enSG744SG744&oq=leow&aqs=chrome.1.69i57j69i59l2j69i61l3.1459j0j7&sourceid=chrome&ie=UTF-8), both the current account and previous account have the same name. Also, his current account forked ab-level1. I.e likely to be him

**Comment by damithc** [comment]
> likely to be him

Yup.

**Pull request merged**